sympl/_components/netcdf.py

```diff
@@ -32,7 +32,7 @@ class NetCDFMonitor(Monitor):
 
         def __init__(
                 self, filename, time_units='seconds', store_names=None,
-                write_on_store=False):
+                write_on_store=False, aliases={}):
```

Comment:
You should set `aliases=None` here, and later check if aliases is None and if so set it to `{}`.

When you do it this way, {} gets instantiated when you create the class, and every class has the same default aliases dictionary. That means if it gets modified in one instance for whatever reason, it gets modified in every instance of that object.